### PR TITLE
Include DBTestSuite for gorm-related tests

### LINKED ISSUE #29: Include DBTestSuite to support gorm in tests

### PR DESCRIPTION
**Pull request by xcoulon**
Fixes #29

### CODE REVIEW

File: test/suite/db_clean.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package cleaner
+package suite
```
Comment (alexeykazakov) [inline test/suite/db_clean.go:1]: Let's also replace db_clean.go by the code we have in Auth. We improved it in Auth too :)
The current (WIT) implementation supports cleaning up records with the only primary key. The advanced version from But Auth also supports composite primary keys (multiple-keys) - https://github.com/fabric8-services/fabric8-auth/blob/master/gormsupport/cleaner/db_clean.go
We use tables with composite primary keys in Auth DB for example.

**Review by alexeykazakov — APPROVED** [review]
LGTM overall. But please replace db_clean.go by the version we have in Auth either in this PR or in a new one.

**Review by dipak-pawar — APPROVED** [review]
Good Job @xcoulon. LGTM

**Review by dipak-pawar — APPROVED** [review]
lgtm @xcoulon.

**Review by dipak-pawar — APPROVED** [review]
Lgtm

**Comment by xcoulon** [comment]
thanks for the review, @dipak-pawar and @alexeykazakov !

**Pull request merged**